### DISCUSSION TITLE
AttributeError: 'function' object has no attribute 'API'

### DISCUSSION THREAD

**Discussion by jauntyd** [tweepy/tweepy#1530] [Questions] [answered]
Hello:

**AttributeError: 'function' object has no attribute 'API'**

Occurs on:
Tweepy 3.10 via Pip on Python 3.7 w/ Django 3.1.6 on FreeBSD 12.2-RELEASE
Tweepy 3.10 via GitHub on Python 3.7 w/ Django 3.1.6 on FreeBSD 12.2-RELEASE

[Traceback](https://bsd.to/FrTM)

If you all need/want any additional info, let me know!

John

**Comment by Harmon758** [comment, marked as answer]
You likely have a function defined as `tweepy` that's overriding the library import.

**Comment by jauntyd** [comment]
Wow. Excellent, my view in DJango was named Tweepy. Perfect, thanks!
